frontend/src/components/dashboard/varianceHomePage/VarianceHomePage.js

```diff
@@ -68,67 +66,69 @@ const propTypes = {
   history: PropTypes.shape({ push: PropTypes.func }).isRequired,
 };
 
-const joinOrRemove = (param, key) => (isEmpty(param) ? {} : { [key]: param.join(",") });
-const formatParams = ({ region = [], compliance_code = [], ...remainingParams }) => ({
-  ...joinOrRemove(region, "region"),
-  ...joinOrRemove(compliance_code, "compliance_code"),
-  ...remainingParams,
-});
-
+export const joinOrRemove = (param, key) => {
+  if (isEmpty(param)) {
+    return {};
+  }
+  return typeof param === "string" ? { [key]: param } : { [key]: param.join(",") };
+};
+export const removeEmptyStings = (param, key) => (isEmpty(param) ? {} : { [key]: param });
```

Comment:
What are all of these changes for?

Reply:
This dropdown selection list was throwing an error when a single item was selected.  It was passing in a string '50' instead of and arrays ['50','39'], the param.join was causing problems.  This checks for a string getting put in